### PR TITLE
Embedding base library module in admin

### PR DESCRIPTION
**Pull request by kobizz**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/pojome/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: core/admin/admin.php [hunk 1]
```diff
@@ -113,11 +124,19 @@ public function enqueue_styles() {
 			'3.8.0'
 		);
 
+		wp_register_style(
```
Comment (KingYes) [inline core/admin/admin.php]: Try embed WP font

File: core/admin/admin.php [hunk 2]
```diff
@@ -717,5 +757,21 @@ public function __construct() {
 
 		// Admin Actions
 		add_action( 'admin_action_elementor_new_post', [ $this, 'admin_action_new_post' ] );
+
+		add_action( 'current_screen', function () {
```
Comment (KingYes) [inline core/admin/admin.php]: space

File: core/admin/admin.php [hunk 3]
```diff
@@ -717,5 +757,21 @@ public function __construct() {
...
+
+	private function init_new_template() {
+		if ( 'edit-elementor_library' === get_current_screen()->id ) {
+			add_action( 'admin_footer', [ $this, 'print_new_template_template' ] );
+
+			add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_new_template_scripts' ] );
+		}
+	}
+
+	private function print_library_layout_template() {
```
Comment (KingYes) [inline core/admin/admin.php]: public

**Review by KingYes — CHANGES_REQUESTED** [review]

**Comment by arielk** [comment]
@kobizz please remove the Google Font loading and sed it again.

**Review by KingYes — APPROVED** [review]

**Pull request merged**